spark/src/main/scala/io/delta/sharing/spark/filters/OpConverter.scala

```diff
@@ -0,0 +1,191 @@
+/*
+ * Copyright (2021) The Delta Lake Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.sharing.spark.filters
+
+import scala.collection.mutable.ListBuffer
+
+import org.apache.spark.sql.catalyst.expressions.{
+  And => SqlAnd,
+  Attribute => SqlAttribute,
+  AttributeReference => SqlAttributeReference,
+  Cast => SqlCast,
+  EqualNullSafe => SqlEqualNullSafe,
+  EqualTo => SqlEqualTo,
+  Expression => SqlExpression,
+  GreaterThan => SqlGreaterThan,
+  GreaterThanOrEqual => SqlGreaterThanOrEqual,
+  In => SqlIn,
+  IsNotNull => SqlIsNotNull,
+  IsNull => SqlIsNull,
+  LessThan => SqlLessThan,
+  LessThanOrEqual => SqlLessThanOrEqual,
+  Literal => SqlLiteral,
+  Not => SqlNot,
+  Or => SqlOr
+}
+import org.apache.spark.sql.types.{
+  BooleanType => SqlBooleanType,
+  DataType => SqlDataType,
+  DateType => SqlDateType,
+  IntegerType => SqlIntegerType,
+  LongType => SqlLongType,
+  StringType => SqlStringType
+}
+
+// A set of utilites that convert SQL expressions to Json predicate Ops.
+//
+// We support this conversion for a subset of SQL expressions, and will
+// throw an exception if the SQL expression cannot be converted.
+//
+// The caller should skip filtering in such cases.
```

Comment:
Nit: This line should be immediately after the previous line? as such cases refer to when SQL expression cannot be converted?

Reply:
Done.